src/models/network.js

```diff
@@ -123,6 +123,22 @@ Network.prototype.getNetworkStatus = function() {
 	return status;
 };
 
+Network.prototype.addChannel = function(newChan) {
+	let index = this.channels.length - 1;
+
+	// Don't sort special channels in amongst channels/users.
+	if (newChan.type === Chan.Type.CHANNEL || newChan.type === Chan.Type.QUERY) {
+		for (let i = 0; i < this.channels.length; i++) {
```

Comment:
Start `i` at 1 to avoid the lobby (and add a comment about it)

Reply:
Done, I tried to add a test for this, but it's not possible as there is no string that comes before empty string.

Reply:
Shouldn't it be possible to create a network called "xxx" (which in turn creates channel lobby) and add a channel named `aaa` to it? That should keep the order.

Reply:
Oh wait, yes, of course, the name of the lobby is coming as '' to me because of course I'm not setting it in the test. Dumb moment.